packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -97,7 +97,18 @@ const ProofOfIdentityContainer = ({
     }
 
     const redirect_button = should_show_redirect_btn && (
-        <Button primary className='proof-of-identity__redirect' onClick={() => routeBackTo(from_platform.route)}>
+        <Button
+            primary
+            className='proof-of-identity__redirect'
+            onClick={() => {
+                if (platforms[from_platform.name?.toLowerCase()].is_hard_redirect) {
```

Comment:
what if platforms[from_platform.name] is null? add a guard for that

Reply:
best to assign `platforms[from_platform?.name]` to a variable and then de-structuring `is_hard_redirect` and `url` out from it. This way you can add a guard by assigning a default value.

```
const platform = platforms[from_platform.name?.toLowerCase()];
const { is_hard_redirect || '', url || '' } = platform;
```